README.md

```diff
@@ -33,7 +33,7 @@ To get a local copy up and running follow these simple example steps.
 > clone the app by running this command
 
 ```bash 
-git clone https://github.com/fobadara/math-magicians-react.git
+git clone https://github.com/fobadara/new-math-magicians-react.git
 ```
```

Comment:
- [ ] I think it would be better if you add an instruction on how to change the directory after cloning the project. For example, cd `project name`.

Reply:
Thank you for the review. I have implemented your suggestion